packages/edges/addon/components/cardstack-edge.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import layout from '../templates/components/cardstack-edge';
+
+const Edge = Component.extend({
+  layout,
+  componentName: computed('edge', function() {
+    return `in-${this.get('edge')}-toolbar`;
+  })
+});
```

Comment:
We can probably reduce this dynamic shenanigans by just extending the base edge component out of ember-toolbars. It's in `ember-toolbars/addon-components/-in-edge`, but that's my addon and we can make it public.